editor/src/components/canvas/controls/select-mode/flex-resize-control.tsx

```diff
@@ -0,0 +1,258 @@
+          ref={bottomRef}
+          position={{ x: 0.5, y: 1 }}
+          cursor={CSSCursor.ResizeNS}
+          direction='horizontal'
+          enabledDirection={DirectionVertical}
+        />
+      </div>
+    )
+  }
+  return null
+})
+
+interface ResizeEdgeProps {
+  cursor: CSSCursor
+  position: EdgePosition
+  direction: 'horizontal' | 'vertical'
+  enabledDirection: EdgePosition
+}
+
+const ResizeEdgeMouseAreaSize = 12
+const ResizeEdge = React.memo(
+  React.forwardRef<HTMLDivElement, ResizeEdgeProps>((props, ref) => {
+    const LineSVGComponent =
+      props.position.y === 0.5 ? DimensionableControlVertical : DimensionableControlHorizontal
+    const dispatch = useEditorState((store) => store.dispatch, 'ResizeEdge dispatch')
+    const jsxMetadataRef = useRefEditorState((store) => store.editor.jsxMetadata)
+    const selectedViewsRef = useRefEditorState((store) => store.editor.selectedViews)
+
+    const onEdgeMouseDown = React.useCallback(
+      (event: React.MouseEvent<HTMLDivElement>) => {
+        startResizeInteraction(
+          event,
+          dispatch,
+          props.position,
+          props.enabledDirection,
+          jsxMetadataRef.current,
+          selectedViewsRef.current,
+        )
+      },
+      [dispatch, props.position, props.enabledDirection, jsxMetadataRef, selectedViewsRef],
+    )
+
+    return (
+      <div ref={ref} style={{ position: 'absolute' }} onMouseDown={onEdgeMouseDown}>
+        <LineSVGComponent />
+        <div
+          style={{
+            position: 'absolute',
+            pointerEvents: 'initial',
+            width: `calc(${ResizeEdgeMouseAreaSize}px / var(--utopia-canvas-scale))`,
+            height: `calc(${ResizeEdgeMouseAreaSize}px / var(--utopia-canvas-scale))`,
+            top: `calc(${-ResizeEdgeMouseAreaSize / 2}px / var(--utopia-canvas-scale))`,
+            left: `calc(${-ResizeEdgeMouseAreaSize / 2}px / var(--utopia-canvas-scale))`,
+            backgroundColor: 'transparent',
+            cursor: props.cursor,
+          }}
+        />
+      </div>
+    )
+  }),
+)
+
+function startResizeInteraction(
+  event: React.MouseEvent<HTMLDivElement>,
+  dispatch: EditorDispatch,
+  position: EdgePosition,
+  enabledDirection: EdgePosition,
+  metadata: ElementInstanceMetadataMap,
+  selectedViews: Array<ElementPath>,
+) {
+  event.stopPropagation()
+  if (event.buttons === 1) {
+    // TODO update this to call createInteractionState to use canvas strategies
+    const centerBasedResize = event.altKey
+    const keepAspectRatio = event.shiftKey // || props.elementAspectRatioLocked ???
+    const enableSnapping = !event.metaKey
+    const canvasPositions = windowToCanvasCoordinatesGlobal(
+      windowPoint({ x: event.nativeEvent.x, y: event.nativeEvent.y }),
+    )
+    const start: CanvasPoint = canvasPositions.canvasPositionRaw
+    const originalFrames = getOriginalFrames(selectedViews, metadata)
+    const isMultiSelect = selectedViews.length !== 1
```

Comment:
I would write `selectedViews.length > 1`
I know it can not be `0` here, but still that is more expressive in my opinion

Reply:
it looks better! fixed